packages/xml-views-validation/test/validators/attributes/unknown-attribute-key-spec.ts

```diff
@@ -0,0 +1,810 @@
+            </mvc:View>`;
+
+        testValidationsScenario({
+          model: ui5SemanticModel,
+          xmlText: xmlSnippet,
+          validators: {
+            attribute: [validateUnknownAttributeKey],
+          },
+          assertion: (issues) => {
+            expect(issues).to.be.empty;
+          },
+        });
+      });
+
+      it("will not detect an issue when the attribute is xmlns (with name)", () => {
+        const xmlSnippet = `
+          <mvc:View
+            xmlns:mvc="sap.ui.core.mvc"">
+            <content xmlns:m="sap.m">
+            </content>
+          </mvc:View>`;
+
+        testValidationsScenario({
+          model: ui5SemanticModel,
+          xmlText: xmlSnippet,
+          validators: {
+            attribute: [validateUnknownAttributeKey],
+          },
+          assertion: (issues) => {
+            expect(issues).to.be.empty;
+          },
+        });
+      });
+    });
+
+    context("unknown tag", () => {
+      it("will not detect an issue when the attribute name is unknown for tag starting with lowecase", () => {
+        const xmlSnippet = `
+          <mvc:View
+            xmlns:mvc="sap.ui.core.mvc"
+            xmlns="sap.m">
+            <content1 unknownattribute="">
+            </content1>
+          </mvc:View>`;
+
+        testValidationsScenario({
+          model: ui5SemanticModel,
+          xmlText: xmlSnippet,
+          validators: {
+            attribute: [validateUnknownAttributeKey],
+          },
+          assertion: (issues) => {
+            expect(issues).to.be.empty;
+          },
+        });
+      });
+
+      it("will not detect an issue when the attribute name is unknown for tag starting with uppercase", () => {
+        const xmlSnippet = `
+          <mvc:View
+            xmlns:mvc="sap.ui.core.mvc">
+            <content>
+              <List unknownattribute=""></List>
+            </content>
+          </mvc:View>`;
+
+        testValidationsScenario({
+          model: ui5SemanticModel,
+          xmlText: xmlSnippet,
+          validators: {
+            attribute: [validateUnknownAttributeKey],
+          },
+          assertion: (issues) => {
+            expect(issues).to.be.empty;
+          },
+        });
+      });
+
+      it("will not detect an issue when the attribute name is unknown for tag in known namespace", () => {
+        const xmlSnippet = `
+          <mvc:View1
+            xmlns:mvc="sap.ui.core.mvc"
+            xmlns="sap.m"
+            busy1="true">
+          </mvc:View1>`;
+
+        testValidationsScenario({
+          model: ui5SemanticModel,
+          xmlText: xmlSnippet,
+          validators: {
+            attribute: [validateUnknownAttributeKey],
+          },
+          assertion: (issues) => {
+            expect(issues).to.be.empty;
+          },
+        });
+      });
+    });
+
+    context("non-reproducible unit tests", () => {
+      it("will not detect an issue when the attribute doesn't have a key", () => {
```

Comment:
may not be worth testing this and instead ignoring this case using `istanbul ignore`

Reply:
I wanted to make sure we don't have unexpected errors